usaspending_api/references/views_v2.py

```diff
@@ -62,6 +62,10 @@ def get(self, request, pk, format=None):
         total_budget_authority_queryset = total_budget_authority_queryset.filter(fiscal_year=active_fiscal_year)
 
         total_budget_authority_submission = total_budget_authority_queryset.first()
+        total_budget_authority_amount = ""
```

Comment:
should we default to 0 or just blank? this should be coordinated with frontend so we know what to display in these kinds of situations